CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 ### Added
+- Technical - Test on postgres from 9.4 to 12.1.
```

Comment:
[Conventions] By convention we accumulate changes by the bottom.
Conventional commit will come to the rescue.